app/src/androidTest/java/org/simple/clinic/di/TestLoginModule.kt

```diff
@@ -1,15 +0,0 @@
-package org.simple.clinic.di
-
-import android.app.Application
-import io.reactivex.Completable
-import org.simple.clinic.login.LoginModule
-import org.simple.clinic.login.LoginOtpSmsListener
-import org.simple.clinic.util.scheduler.SchedulersProvider
-
-class TestLoginModule : LoginModule() {
```

Comment:
Curious, why are we removing this?

Reply:
This was used to replace only the `LoginOtpSmsListener` with a no-op version in tests. We broke this out into a separate module which needs to be included separately, so this can be removed from the dagger graph in tests because it is not needed.